libs/ai-endpoints/tests/unit_tests/test_api_key.py

```diff
@@ -43,3 +44,13 @@ def get_api_key(instance: Any) -> str:
         assert get_api_key(public_class(nvidia_api_key="PARAM")) == "PARAM"
         assert get_api_key(public_class(api_key="PARAM")) == "PARAM"
         assert get_api_key(public_class(api_key="LOW", nvidia_api_key="HIGH")) == "HIGH"
+
```

Comment:
we also need a test where a session.post is made and the last_input is validated to not include the secret key